test/pjrt/test_dynamic_plugin_tpu.py

```diff
@@ -0,0 +1,34 @@
+import concurrent.futures
+
+from absl.testing import absltest
+import torch_xla.core.xla_model as xm
+import torch_xla.distributed.xla_multiprocessing as xmp
+from torch_xla.experimental import plugins
+import torch_xla.runtime as xr
+from torch_xla._internal import tpu
+
+plugins.register_plugin('TPU', tpu.TpuPlugin())
+plugins.use_dynamic_plugins()
+
+
+class TestDynamicTpuPlugin(absltest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    xr.set_device_type('TPU')
+
+  @staticmethod
+  def _assert_tpus_exist(index=0):
+    del index
```

Comment:
what's this for?

Reply:
`index` is required for `spawn`, but we don't need it. I just explicitly delete it to mark it unused